storage/container_storage_interface/persistent-storage-csi-vsphere.adoc

```diff
@@ -51,7 +51,33 @@ include::modules/persistent-storage-csi-vsphere-install-issues.adoc[leveloffset=
 include::modules/persistent-storage-csi-vsphere-top-aware.adoc[leveloffset=+1]
 [role="_additional-resources"]
 .Additional resources
-* https://docs.vmware.com/en/VMware-vSphere/8.0/vsphere-vcenter-esxi-management/GUID-16422FF7-235B-4A44-92E2-532F6AED0923.html?hWord=N4IghgNiBcIC5gOYgL5A[VMware vSphere tag documenation]
+* https://docs.vmware.com/en/VMware-vSphere/8.0/vsphere-vcenter-esxi-management/GUID-16422FF7-235B-4A44-92E2-532F6AED0923.html?hWord=N4IghgNiBcIC5gOYgL5A[VMware vSphere tag documentation]
 
+[id="vsphere-pv-encryption"]
+== vSphere persistent disks encryption
+
+You can encrypt virtual machines (VMs) and dynamically provisioned persistent volumes (PVs) on {product-title} running on top of vSphere. 
+
+You must encrypt VMs before you can encrypt PVs, which you can do during installation or post-installation. 
```

Comment:
we need to mention that we don't support RWX encrypted PVs. you can't request a RWX PVs out of a SC that uses an encrypted storage policy
@gnufied